Userland/Libraries/LibGfx/ImageFormats/GIFWriter.cpp

```diff
@@ -147,4 +188,13 @@ ErrorOr<void> GIFWriter::encode(Stream& stream, Bitmap const& bitmap)
     return {};
 }
 
+ErrorOr<NonnullOwnPtr<AnimationWriter>> GIFWriter::start_encoding_animation(SeekableStream& stream, IntSize dimensions)
+{
+    TRY(write_header(stream));
+
```

Comment:
Do you want a FIXME for writing the loop count?

Reply:
I'm good the PR is already waiting :smiley: